### ISSUE TITLE
Version 4.0.0 does not support Angular 16 

### ISSUE THREAD

**Issue by henkkelder** [ckeditor/ckeditor4-angular#249]
It appears the just released version 4.0.0 does not support Angular 16 :-(

`This likely means that the library (ckeditor4-angular) which declares CKEditorModule is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy. `

**Comment by Comandeer** [comment]
@henkkelder, could you recheck? I've prepared a [sample application with Angular 16](https://stackblitz-starters-svxhrm.stackblitz.io) and it seems to work.

**Comment by henkkelder** [comment]
The link to your sample applicaties does seem to go anywhere. I see only this : `{"originalErr":{}}`

But I tried it a couple of days ago and this morning I tried it again. Result was the same. 

Somehow the difference is in how the distribution is build. I had cloned your repository and made a build that worked ok.
The distributable version I have build creates ckeditor.module.d.ts as:

```
import * as i0 from "@angular/core";
import * as i1 from "./ckeditor.component";
import * as i2 from "@angular/forms";
import * as i3 from "@angular/common";
export declare class CKEditorModule {
    static ɵfac: i0.ɵɵFactoryDeclaration<CKEditorModule, never>;
    static ɵmod: i0.ɵɵNgModuleDeclaration<CKEditorModule, [typeof i1.CKEditorComponent], [typeof i2.FormsModule, typeof i3.CommonModule], [typeof i1.CKEditorComponent]>;
    static ɵinj: i0.ɵɵInjectorDeclaration<CKEditorModule>;
}
export * from './ckeditor';
export { CKEditorComponent } from './ckeditor.component';
```

Your version is:
```/**
 * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
 * For licensing, see LICENSE.md.
 */
export declare class CKEditorModule {
}
export * from './ckeditor';
export { CKEditorComponent } from './ckeditor.component';
```

**Comment by Comandeer** [comment]
@henkkelder, ok, thanks, now I see the issue. We will look into it.

**Comment by jacekbogdanski** [comment]
@henkkelder it looks like our building/publishing environment tricked us a bit this time and used the wrong dependencies for the bundle package. We are going to release a patch fix soon to address that issue. I apologise for the inconvenience!

**Comment by jacekbogdanski** [comment]
@henkkelder we fixed the issue with a quick 4.0.1 hotfix. It can be confirmed by opening this demo sample that uses Angular v16 - https://codesandbox.io/p/sandbox/winter-leaf-878tn5
